.github/workflows/test.yml

```diff
@@ -100,6 +100,8 @@ jobs:
             subset: singleuser
           - python: "3.11"
             selenium: selenium
```

Comment:
let's use `browser: browser` instead of an implementation-specific name

Reply:
thank you, the changes have been done